week-2-wide-perspective/w02e05-simple-server-side/src/types/user.ts

```diff
@@ -0,0 +1,3 @@
+import { User } from '@prisma/client'
```

Comment:
To mi tutaj nie pasuje totalnie. Do tej pory robiłeś podział `per feature` więc to powinno w mojej opinii mieszkać w `/users`